test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -1269,6 +1269,11 @@ class InheritedClassTest : System.Attribute
             $res.CompletionMatches[0].CompletionText | Should -BeExactly $afterTab
         }
 
+        It "Tab completion UNC path with filesystem provider" -Skip:(!$IsWindows) {
```

Comment:
Since your regex explicit avoids device paths, should have a test for that.

Reply:
I couldn't test it because the wildcard globbing was broken (see: https://github.com/PowerShell/PowerShell/pull/19442 ) but now that it has been fixed, I'll add the test.